test/functional/page_objects/dashboard_page.ts

```diff
@@ -174,9 +174,13 @@ export class DashboardPageObject extends FtrService {
     await this.testSubjects.existOrFail('dashboardLandingPage');
   }
 
-  public async clickDashboardBreadcrumbLink() {
+  public async clickDashboardBreadcrumbLink(isServerless: boolean = false) {
```

Comment:
Instead of props, how about separate methods for serverless.
1) clickBreadcrumbsToGotoLandingPage
2) serverlessClickBreadcrumbsToGotoLandingPage

And how about moving clickDashboardBreadcrumbLink functionality into goto method. Its not used anywhere else and does not really need to be a separate method.



So `clickServerlessDashboardcrumbLink`

Reply:
Good call, I've cleaned that up a bit in [b312967](https://github.com/elastic/kibana/pull/164992/commits/b312967f61c024a8c359161f7f0e7c4781ffb80b)

Reply:
Nit: lens calls this method `goToListingPageViaBreadcrumbs`. I think it would be best to use a name like this that inlcudes breadcrumbs since other page objects have gotoListPages that just use `navigateToUrlWithBrowserHistory` to load listing page. For example [maps.gotoMapListingPage](https://github.com/elastic/kibana/blob/main/x-pack/test/functional/page_objects/gis_page.ts#L261)

Reply:
That's not a bad idea! It's used in many places so I'll hold off on renaming it in this PR to not trigger a bunch more reviewers.